app/src/main/java/org/merakilearn/datasource/ApplicationRepo.kt

```diff
@@ -81,6 +84,14 @@ class ApplicationRepo(
 
     suspend fun fetchClassData(classId: Int): Classes? {
         return try {
+
+            // To disable crashylytics in Debug application mode
+            if(BuildConfig.DEBUG) {
```

Comment:
This should be done in Application class.

Reply:
got it, I'll create new PR for this.

Reply:
This is done now @suhaibroomy, You can pick latest commit. 

But not in new PR, I'll take care from next time separate work, separate PRs.